app/setup.py

```diff
@@ -325,6 +326,20 @@ def setup_babel(application):
     application.babel = Babel(application)
     application.jinja_env.add_extension('jinja2.ext.i18n')
 
+    @application.babel.localeselector
+    def get_locale():  # pylint: disable=unused-variable
+        session = get_session_store()
+
+        if not session:
+            return None
```

Comment:
Shall we default to 'en' here?

Reply:
Returning `None` tells babel to use it's global default, which is `en`.